src/components/TokenBottomSheet.tsx

```diff
@@ -325,40 +328,38 @@ function TokenBottomSheet({
           />
         )}
       </View>
-    </>
+    </View>
   )
 
   return (
     <>
       {isScreen ? (
-        <View
-          // use fixed height if there are filter chips, otherwise the bottom
-          // sheet height changes as tokens as filtered
-          style={filterChips.length ? styles.screenContainerFixed : styles.screenContainer}
-          testID="TokenBottomSheet"
-        >
-          {content}
-        </View>
+        // Don't wrap the content in a BottomSheetBase when used as a screen
+        // since the bottom sheet navigator already provides the necessary wrapping
+        content
       ) : (
-        <BottomSheetBase forwardedRef={forwardedRef} snapPoints={snapPoints}>
-          <View style={styles.container} testID="TokenBottomSheet">
+        <BottomSheetModalProvider>
```

Comment:
We need one here because of the nested bottom sheets. It would be cleaner to remove the TokenBottomSheet that acts as a screen, but that can be in a follow-up PR.

Reply:
Removed in 3f60c98510f1145d4a99a262dce8276a3dc92a73. We needed one to wrap the `NetworkMultiSelectBottomSheet` because of the double bottom sheets in the buy flow; however, the one wrapping the `BottomSheetBase` component was not needed.

Reply:
Reverted the removal in 6ba328fc23309fe5c21873ebbf4e7def362a745c. We do need this for the swap screen. Otherwise, the network selection renders behind the TokenBottomSheet.